data/debug-cli/docker_debug.yaml

```diff
@@ -1,5 +1,5 @@
 command: docker debug
-short: Get a shell into any container or image. Replacement for `docker exec`.
+short: Get a shell into any container or image. An alternative to debugging with `docker exec`.
```

Comment:
So the main reason to bring `docker exec` up here at all is because it makes people who know `docker exec` understand `docker debug` very quickly. But it's true that feature wise it's not an alternative/replacement/.. It's a different tool that is very similar to `docker exec` but can be applied to a larger set of problems and does a lot of things very different. Maybe we should go that route?

Eg., something like "Similar as `docker exec` but without the limitations as it can be used on any container and images plus it comes with lots of tools included already." (you probably know of better ways to say this)

WDYT? Like the point is not about comparing them but showing that users can do more now.